index.html

```diff
@@ -139,27 +135,22 @@ <h5 class="display-3 text-center project-stat">
                             <img src="assets/img/projects/irc.png" class="card-img-top project-img img-center" alt="IRC"
                             >
                             <p class="card-text text-darker text-center">The International Research Council (IRC) is a
-                                community
-                                created by
-                                SEF to
-                                showcase the work in various fields of Sri Lankan researchers.</p>
-                            <h5 class="display-3 text-center project-stat">100+</h5>
+                                community created by SEF to showcase the work in various fields of Sri Lankan researchers.</p>
+                            <h5 class="display-3 text-center project-stat" id="irc-stat">100+</h5>
                             <p class="text-dark text-center stats">
                                 researchers of Sri Lankan origin representing the world's top universities
                             </p>
                         </div>
                     </a>
-		    <a href="onelive" class="card col-lg-4 shadow project-card">
+		        <a href="onelive" class="card col-lg-4 shadow project-card">
```

Comment:
Can you remove this space? @RamithaHeshan33 its not still removed, it should be like the line 158